ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -101,6 +121,26 @@ class GameOptionsViewController: UIViewController {
         // Pass the selected object to the new view controller.
     }
     */
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // do some customization
+        
+        let gameRoom = GameRoom()
+        gameRoom.current_question = 0
+        // this needs to be retrieved from create game
+        gameRoom.is_public = true
+        gameRoom.max_num_of_people = 10
```

Comment:
I'm not 100% sure when this is used because I didn't work on it. I think I included this when I tried to resolve conflicts. 

If removing this section is fine, then I'll go ahead and remove it.